release notes/v0.40.0.md

```diff
@@ -0,0 +1,270 @@
+k6 v0.40.0 is here! :tada: This release includes the promised experimental modules, some other small fixes and a lot of changes needed for the metric time series, unindexable tags and native ESM support. [Roadmap and future plans section](#roadmap-and-future-plans) makes another appearance with more info on how things are going and what should be expected
+
+# Breaking changes
+
+- [#2632](https://github.com/grafana/k6/pull/2632) As part of refactoring to set tags provided to `metric.add` in the order they are provided, it was found out that you can provide tags as multiple objects/maps. This has never been the intend use and has never been documented. As such and as that makes no sense given every other API we have decided to remove this ability.
+- [#2582](https://github.com/grafana/k6/pull/2582) If `RegisterCallback` result is called twice this will now lead to a panic, instead of the second call silently breaking the event loop. This has never been the expected behavior and calling it twice is always a bug in the code using it.
+- As part of [#2596](https://github.com/grafana/k6/pull/2596) the `tainted` property of the Metric type is no longer outputted by the JSON output. That property was (likely) always going to have the value `false` as it was outputted at the beginning of the test.
```

Comment:
```suggestion
- [#2632](https://github.com/grafana/k6/pull/2632) During the refactoring to set tags to `metric.add` in the order they are provided, we discovered that you can provide tags as multiple objects or maps. This has never been the intended use and was never documented. As it was undocumented, and as such functionality makes no sense alongside every other API k6 has, we decided to remove this ability.
- [#2582](https://github.com/grafana/k6/pull/2582) Previously, if `RegisterCallback` result was called twice, the second call would silently break the event loop. This has never been expected behavior, and calling it twice is always a bug in the code using it. Now, calling the `RegisterCallback` result twice leads to a panic.
- [#2596](https://github.com/grafana/k6/pull/2596) The `tainted` property of the Metric type is no longer outputted by the JSON output. That property was likely always going to have a `false` value as it was outputted at the beginning of the test.
```

Reply:
> objects or map

the thing is that object and map here is the same thing ... I am not saying you can do one or the other, but using two words to refer to the same thing. Should in that case still use `objects/map` or maybe just leave one of it :thinking:

Reply:
I'd just choose "object." I must confess that I've always been confused by what a "map" is, so I'm happy to discover that at last, like everything else, it's just a JS object.

 I think `/` suggests an either-or situation most of the time, as in `either/or` or `true/false` or `pass/fail`. For that reason, I don't like using it as a shorthand for a conjunction.
 
 If you want to make the objects-therefore-maps implication clear, you could use a parenthetical:
 
 >   we discovered that you can provide tags as multiple objects (and therefore maps)